app/wama.web/Views/CheckIn/Index.cshtml

```diff
@@ -23,11 +23,20 @@
             <div class="form-group col-xs-12">
                 <input type="submit" class="btn btn-lg btn-primary big-action-button" value="Check In" />
             </div>
-
             <br />
             <br />
             <br />
-            <a href='@Url.Content("~/Patron/Create")'>create new account</a>
```

Comment:
At this point before rendering this link, you could check that value in the ViewBag OR ViewData